circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -223,7 +220,7 @@ constexpr size_t MAX_NOTES_PER_PAGE = 10;
 constexpr size_t VIEW_NOTE_ORACLE_RETURN_LENGTH = MAX_NOTES_PER_PAGE * (MAX_NOTE_FIELDS_LENGTH + 1) + 2;
 
 constexpr size_t CALL_CONTEXT_LENGTH = 6;
-constexpr size_t COMMITMENT_TREES_ROOTS_LENGTH = 5;
+constexpr size_t BLOCK_DATA_LENGTH = 7;
```

Comment:
can you write a comment here so that if people add any more trees to the block tree, they can easily find this and remember to update it?